airgun/entities/container.py

```diff
@@ -37,4 +70,26 @@ class AddNewContainer(NavigateStep):
     prerequisite = NavigateToSibling('All')
 
     def step(self, *args, **kwargs):
-        self.parent.browser.click(self.parent.new)
+        self.parent.new.click()
+
+
+@navigator.register(ContainerEntity, 'Details')
+class ContainerDetails(NavigateStep):
+    """Navigate to container details screen"""
+    VIEW = ContainerDetailsView
+
+    def prerequisite(self, *args, **kwargs):
+        return self.navigate_to(self.obj, 'All')
+
+    def am_i_here(self, *args, **kwargs):
+        entity_name = kwargs.get('entity_name')
+        return (
+            self.view.is_displayed
+            and self.view.breadcrumb.read() == '{} - /{}'.format(
```

Comment:
strange construct =)

Reply:
You don't say :)
That's how it looks on WebUI:
![image](https://user-images.githubusercontent.com/11719030/53810299-4eeb5780-3f5f-11e9-9e36-e63d19bb3c4f.png)